pom.xml

```diff
@@ -0,0 +1,65 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+ 
+  <groupId>org.exoplatform.maven.plugin</groupId>
+  <artifactId>diff-extension-maven-plugin</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>8</source>
+          <target>8</target>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+  <packaging>maven-plugin</packaging>
+ 
+  <name>Diff Extension Maven Plugin</name>
+  <properties>
+    <mavenVersion>3.0</mavenVersion>
+  </properties>
+ 
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-plugin-api</artifactId>
+      <version>${mavenVersion}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-core</artifactId>
+      <version>${mavenVersion}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.maven.shared</groupId>
+      <artifactId>maven-artifact-transfer</artifactId>
+      <version>0.11.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.codehaus.plexus</groupId>
+      <artifactId>plexus-archiver</artifactId>
+      <version>4.1.0</version>
+    </dependency>
+ 
+    <!-- dependencies to annotations -->
+    <dependency>
+      <groupId>org.apache.maven.plugin-tools</groupId>
+      <artifactId>maven-plugin-annotations</artifactId>
+      <version>3.4</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.googlecode.java-diff-utils</groupId>
+      <artifactId>diffutils</artifactId>
+      <version>1.2</version>
+      <type>jar</type>
+      <scope>compile</scope>
+    </dependency>
+
+  </dependencies>
+</project>
```

Comment:
Should we add the eXo maven repository here to make it easier for developers (no additional steps to make it work) ?

Reply:
which repository do you mean ?

Reply:
The exo repository, like we did on addons, for example for task: https://github.com/exo-addons/task/blob/develop/pom.xml#L146-L173

Reply:
what is the aim of this ?
As I understand, it is allow someone to build the project, without having repository declared in his maven settings, rigth ?
So, this should be at upper level, no ? in this artifact for example :
<artifactId>addons-parent-pom</artifactId>
    <groupId>org.exoplatform.addons</groupId>
    <version>12-M01</version>

Reply:
> As I understand, it is allow someone to build the project, without having repository declared in his maven settings, rigth ?
Right.

> So, this should be at upper level, no ?
Yes we can but it requires to inherit from the addon parent pom, which is not the case here.
But not a big deal, it is such a suggestion to improve dev experience of potential external contributors.

Reply:
done